### DISCUSSION TITLE
sysbuild: free standing configurations

### DISCUSSION THREAD

**Discussion by arifbalik** [zephyrproject-rtos/zephyr#83420] [Q&A] [answered]
I have two apps that I want to configure and build with sysbuild. The apps are under `apps/app1` and `apps/app2` and I want to include `sysbuild.conf`, `sysbuild.cmake` and the `sysbuild` folder under `apps`, but of course I can not run west on `apps` and sysbuild wont pick up any config under `apps` if I run west on `app1` or `app2`

What are your suggestions in this case. I also have mcuboot enabled for both app so a single sysbuild.conf would be very nice.

```
apps
├── app1
│   └── ...
├── app2
│   └── ...
├── sysbuild.conf
├── sysbuild.cmake
└── sysbuild
    ├── app1.conf
    ├── app2.conf
    └── mcuboot.conf
```

**Comment by nordicjm** [comment, marked as answer]
Not supported, you need a primary image. Note that sysbuild is designed for building one project, not for building multiple at the same time, so one project might consist of multiple images for a single device but should not really consist of 2 completely independent and different applications that e.g. occupy the same regions in memory (although building will still work for that, it is just not designed for that usecase)
